seed/views/v3/uploads.py

```diff
@@ -332,7 +332,7 @@ def create_from_pm_import(self, request):
                 pm_csv_writer.writerow(row)
 
         # Look up the import record (data set)
-        org_id = request.query_params.get('organization_id', None)
+        org_id = request.data['organization_id']
```

Comment:
I think we want to keep this as `org_id = request.query_params.get('organization_id', None)`. During the v3 transition we tried to move the organization id out of request payloads into the query params for consistency.

Reply:
Ok wasn't aware the change. Just to confirm that if org_id is missing it doesn't work and it is expected.

Reply:
Yes that's correct

Reply:
Whoopsies, After more testing we should actually have it the way you made it @lainsworth8801  -- I changed it back, sorry for the confusion!